tests/cuda/CMakeLists.txt

```diff
@@ -14,6 +14,27 @@ include( vecmem-compiler-options-cuda )
 # External dependency/dependencies.
 find_package( CUDAToolkit REQUIRED )
 
+add_library(
+   vecmem_testing_cuda_main
+   SHARED
+   test_cuda_main.cu
+)
+
+target_link_libraries(
+   vecmem_testing_cuda_main
+   PUBLIC
+   CUDA::cudart
+   gtest
+   vecmem::core
+   vecmem::cuda
+)
+
+add_library(
+   vecmem::testing_cuda_main
+   ALIAS
+   vecmem_testing_cuda_main
+)
```

Comment:
I know that these aliases are created in the other Acts projects. But I don't see a reason for them existing.

Aliases should be there to provide user friendly names for the "public" libraries of a project. To make it possible to use for instance `vecmem::core` with that name regardless of whether it is picked up pre-compiled, or is being built as part of the current project.

This library should only be used by these tests, and nobody else. So setting up the alias seems to be just a waste of a few lines of code. (This is why `vecmem_testing_common` has the name that it does...)